testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java

```diff
@@ -685,6 +686,7 @@ public static void assertRealm(RealmRepresentation realm, RealmRepresentation st
 
     @Test
     public void clearRealmCache() {
+        Assume.assumeTrue("Realm cache disabled.", isRealmCacheEnabled());
```

Comment:
Just out of curiosity, it is not necessary to fix this. Is there some simple summary of why the tests need this?